src/commands/cicdPipeline.ts

```diff
@@ -0,0 +1,53 @@
+import * as vscode from 'vscode';
+import * as k8s from 'vscode-kubernetes-tools-api';
+import * as utils from 'util';
+import { IActionContext } from 'vscode-azureextensionui';
+
+const deployToAzureExtensionId = 'ms-vscode-deploy-azure.azure-deploy';
+const configurePipelineCommand = 'configure-cicd-pipeline';
+const browsePipelineCommand = 'browse-cicd-pipeline';
+
+export async function configurePipeline(context: IActionContext, target: any): Promise<void> {
+    if (await isDeployToAzureExtensionInstalled()) {
+        await getClusterAndExecute(target, configurePipelineCommand);
+    }
+}
+
+export async function browsePipeline(context: IActionContext, target: any): Promise<void> {
+    if (await isDeployToAzureExtensionInstalled()) {
+        await getClusterAndExecute(target, browsePipelineCommand);
+    }
+}
+
+async function getClusterAndExecute(target: any, command: string): Promise<void> {
+    const cloudExplorer = await k8s.extension.cloudExplorer.v1;
+    if (!cloudExplorer.available) {
+        return undefined;
+    } else {
```

Comment:
💡 Just an idea: (I feel `else` is no longer in need?)

sample Pseudo code is:

```
if (!available) {
    return undefined;
}

// do stuff because I have the stuff available? so I guess else is not longer in need here? 

```